### DISCUSSION TITLE
Wrong scrollView content position with custom sheet positions

### DISCUSSION THREAD

**Discussion by grenos** [LucasMucGH/BottomSheet#78] [Q&A] [answered]
Hey Lucas,

awesome library! 🎉

I've noticed a strange behaviour with the scrollView (_both the one implemented by you and the SwiftUI one_) when i'm using custom sheet positions.  Any idea what might be causing this? Maybe it's my implementation?

https://user-images.githubusercontent.com/26184688/126534340-7e61195f-6126-41cc-bc19-46b45c64caa2.mp4

**Comment by lucaszischka** [comment, marked as answer]
Hello i would guess that you are using an enum something like this:

```swift
enum CustomBottomSheetPosition: CGFloat, CaseIterable {
    case middle = 0.5, top = 0.8
}
```

As described in the documentation, the lowest value (in your case `.middle`) gets the behaviour of the `.bottom` position (hiding the main content).

In your case the only thing you need to change is to either add another position at the height of e.g. 0.2 or disabling this behaviour altogether by adding `.noBottomPosition` to your options.

If this doesn't work for you, please share your source code (only the custom BottomPosition enum and the .bottomSheet part) and I will reopen this.
